### DISCUSSION TITLE
Leaflet popup/tooltip methods are not supported?

### DISCUSSION THREAD

**Discussion by cyberNKYX** [zauberzeug/nicegui#3992] [Q&A] [answered]
### Description

methods like "bindTooltip", "bindPopup" on leaflet layers are not supported. Tried to run these methods on marker or generic layers but nothing happens.

**Comment by falkoschindler** [comment, marked as answer]
Here is an example demonstrating both `bindTooltip` as well as `bindPopup`:
```py
@ui.page('/')
async def page():
    m = ui.leaflet(center=(51.5, 0.0))
    await m.initialized()
    marker = m.marker(latlng=(51.5, 0.0))
    m.run_layer_method(marker.id, 'bindTooltip', 'Marker')
    marker = m.marker(latlng=(51.5, 0.01))
    m.run_layer_method(marker.id, 'bindPopup', 'Popup')
```
<img width="578" alt="Screenshot 2024-11-15 at 09 03 27" src="https://github.com/user-attachments/assets/31e4e593-ed35-4954-a016-78bdff971ff0">

**Comment by cyberNKYX** [comment]
It seems I omitted this line: `await m.initialized()`, looks like I have to wait the element to be initialized before adding interactions. Thank you very much!
